appserver/admingui/plugin-service/src/main/java/org/glassfish/admingui/plugin/ConsolePluginService.java

```diff
@@ -128,17 +130,14 @@ protected synchronized void init() {
 //System.out.println("IP Map: " + pointsByType.toString());
 
         // Log some trace messages
-        if (logger.isLoggable(Level.FINE)) {
-            logger.fine("Console Plugin Service has been Initialized!");
-            if (logger.isLoggable(Level.FINEST)) {
```

Comment:
toString is here resolved before the log method is invoked. It would be better to join both lines into 
```
logger.log(Level.CONFIG, "Console Plugin Service has been initialized. Integration points by type: \n{0}", pointsByType);
```